app/models/question.rb

```diff
@@ -0,0 +1,3 @@
+class Question < ApplicationRecord
+  has_many :answers
```

Comment:
Same as above.

Reply:
I added length and presence, should I add more

Reply:
Should be good for now